### PR TITLE
Create docker-compose.yml

### PR DESCRIPTION
**Pull request by deveth0**
Minor change that adds a docker-compose file to simplify deployments

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: "3"
+
+services:
+  fritzdocsis:
+    container_name: fritzdocsis
+    build: .
+    ports:
+      - "2112:2112/tcp"
+    restart: unless-stopped
+    entrypoint: ["./fritzDocsis", "-url", "http://192.168.0.1", "-username", "YOUR_USER_NAME", "-password", "YOUR_PASSWORD"]
```
Comment (mulbc) [inline docker-compose.yml]: I'm not a big fan of docker-compose, but I know it is popular, so I will add it if you change the url parameter. Either remove it completely to default to `192.168.178.1` or set it in here to that value.
`192.168.178.1` is the default FritzBox IP and is the address that most people will have their box set to.

**Review by mulbc — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (deveth0) [reply docker-compose.yml]: http://fritz.box would be even better, cause this also resolves, if you change the ip

Comment (mulbc) [reply docker-compose.yml]: That would be fine with me too

**Review by mulbc — APPROVED** [review]
lgtm now - thanks

**Pull request merged**